persistence/src/commonMain/db_user/com/wire/kalium/persistence/Conversations.sq

```diff
@@ -253,6 +253,9 @@ SELECT * FROM ConversationDetails WHERE mls_group_state = ? AND (protocol IS "ML
 getConversationIdByGroupId:
 SELECT qualified_id FROM Conversation WHERE mls_group_id = ?;
 
+selectConversationIdsByProtocol:
```

Comment:
even self and 1:1 conversations?

Reply:
No, self and 1:1 conversations are unnecessary because they never change protocol. I'll exclude them.

Reply:
@typfel, could you please add tests for this too?